extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -0,0 +1,98 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import { Product } from '../models/product';
+import { CONGRATULATIONS, SKU_RECOMMENDATION_PAGE_TITLE, SKU_RECOMMENDATION_ALL_SUCCESSFUL } from '../models/strings';
+
+export class SKURecommendationPage extends MigrationWizardPage {
+	// For future reference: DO NOT EXPOSE WIZARD DIRECTLY THROUGH HERE.
```

Comment:
Why?

Reply:
just to maintain cleaner code, if something needs to happen with the wizard object itself it should be abstracted out to the state model (its just for my own reference in the future) a lot of this code is temporary :D

Reply:
If you plan on taking this out that's fine - but if this is going to stay in then I'd suggest adding a bit of context here